packages/babel-code-frame/src/index.js

```diff
@@ -190,7 +190,7 @@ export function codeFrameColumns(
     (opts.highlightCode && Chalk.supportsColor) || opts.forceColor;
   let chalk = Chalk;
```

Comment:
Totally not in the scope of this PR, but wouldn't this work as a ternary?
```
const chalk = opts.forceColor ? new Chalk.constructor({ enabled: true, level: 1 }) : Chalk;
```

Reply:
Sure.